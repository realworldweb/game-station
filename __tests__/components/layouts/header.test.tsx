import { render } from '@testing-library/react';
import Header from '@/components/layout/header';

describe('<Header />', () => {
	it('should render', () => {
		const { container } = render(<Header />);
		expect(container).toMatchSnapshot();
	});
});
