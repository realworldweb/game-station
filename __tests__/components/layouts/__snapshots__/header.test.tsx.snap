// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render 1`] = `
<div>
  <div
    class="relative flex flex-none flex-wrap w-100 h-14 content-center justify-center bg-neutral-900 text-white"
    title="header"
  >
    <p
      class="text-center"
    >
      The Game Station
    </p>
  </div>
</div>
`;
